src/Core/Migration/V6_7/Migration1740563553AddAppRequestedPrivileges.php

```diff
@@ -24,6 +24,8 @@ public function update(Connection $connection): void
             'app',
             'requested_privileges',
             'json',
+            false,
+            '(JSON_ARRAY())'
```

Comment:
```suggestion
            '[]'
```

Reply:
causes a syntax error:
`Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[]' at line 1`